gammapy/datasets/tests/test_map.py

```diff
@@ -1909,3 +1909,11 @@ def test_peek(images):
 def test_create_psf_reco(geom):
     dat = MapDataset.create(geom, reco_psf=True)
     assert isinstance(dat.psf, RecoPSFMap)
+
+
+def test_stat_sum():
+    axis = MapAxis.from_energy_bounds(0.1, 10, 5, unit="TeV")
+    geom = RegionGeom.create(None, axes=[axis])
+    dataset = SpectrumDatasetOnOff.create(geom)
+    dataset.counts_off = None
+    return dataset.stat_sum()
```

Comment:
```suggestion
    stat = dataset.stat_sum()
    assert stat == 0
```
Actually a test should assert something. (and run obviously).